libavfilter/dnn/dnn_backend_native_layer_mathbinary.c

```diff
@@ -27,6 +27,79 @@
 #include "libavutil/avassert.h"
 #include "dnn_backend_native_layer_mathbinary.h"
 
+typedef float (*FunType)(const float src, float dst);
+FunType Pfun;
+
+static float sub(float src, float dst)
+{
+    return src - dst;
+}
+static float add(float src, float dst)
+{
+    return src + dst;
+}
+static float mul(float src, float dst)
+{
+    return src * dst;
+}
+static float realdiv(float src, float dst)
+{
+    return src / dst;
+}
+static float minimum(float src, float dst)
+{
+    return FFMIN(src, dst);
+}
+static float floormod(float src, float dst)
+{
+    return (float)((int)(src) % (int)(dst));
+}
+
+static void mathbin1(FunType Pfun, const DnnLayerMathBinaryParams *params, const DnnOperand *input, DnnOperand *output, DnnOperand *operands, const int32_t *input_operand_indexes)
```

Comment:
use a meaningful name for mathbin1 and mathbin2

Reply:
> use a meaningful name for mathbin1 and mathbin2

Has been adopted.